api/explorer.go

```diff
@@ -20,6 +22,8 @@ type (
 		MinerPayoutIDs []types.SiacoinOutputID `json:"minerpayoutids"`
 		Transactions   []ExplorerTransaction   `json:"transactions"`
 		RawBlock       types.Block             `json:"rawblock"`
+		HexBlock       string                  `json:"hexblock"`
+		BlockID        string                  `json:"blockid"`
 
 		modules.BlockFacts
```

Comment:
hmm, changing to `BlockID` actually caused tests to start failing, because `modules.BlockFacts` has an embedded `BlockID` field.

Is it necessary to have a `BlockID string` field if we already have it as a `[32]byte`? If so, maybe call it `HexBlockID`?

Reply:
@lukechampine  when i remove the field i get:
```
➜  Sia git:(explorer-patch) ✗ make
go install -tags 'netgo' -ldflags='-s -w' ./api ./build ./compatibility ./crypto ./encoding ./modules ./modules/consensus ./modules/explorer ./modules/gateway ./modules/host ./modules/host/contractmanager ./modules/renter ./modules/renter/contractor ./modules/renter/hostdb ./modules/renter/hostdb/hosttree ./modules/renter/proto ./modules/miner ./modules/wallet ./modules/transactionpool ./persist ./siac ./siad ./sync ./types
# github.com/NebulousLabs/Sia/api
api/explorer.go:201: unknown field 'BlockID' in struct literal of type ExplorerBlock
Makefile:75: recipe for target 'release-std' failed
make: *** [release-std] Error 2
```
therefore i renamed it to HexBlockID however i left the json named the standard way because to me it seems cleaner. please feel free to adjust again if you think there is a cleaner way to handle this.